startup.sh

```diff
@@ -3,12 +3,12 @@
 
 export ACCESS_LOG_FORMAT='%(h)s %(l)s %({REMOTE_USER}i)s %(t)s "%(r)s" "%(q)s" %(s)s %(b)s %(D)s'
 
+export LCLS_NAMING_TOOL_TAXONS_DIR="$(pwd)/lcls_naming_tool/taxons"
```

Comment:
This only works if you run this script from the lcls_naming_tool repo.  You'll need to grab the location of this file specifically to make this fully robust.  Something like:
```
SCRIPT_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" || return pwd -P)
```
I have this in one of my scripts, but maybe stack overflow has a better solution

Reply:
I think startup.sh is used only for the web app. Should dirname be set to the absolute file path to /lcls_naming_tool on the pswww server?

Reply:
A related thought: we shouldn't be doing any of these because the default taxons path should already pick the local repo.

If the goal is to link to the most-up-to-date taxons for the prod server this should really be
```suggestion
export LCLS_NAMING_TOOL_TAXONS_DIR="/cds/group/pcds/pyps/apps/lcls_naming_tool/taxons"
```

Reply:
`dirname` is a bash command.  The command I posted above intends to return the full location of the script that's being run, so you don't have to hard code it.  The problem with the original is that it's dependent on where you run the script from:
```bash
(base)roberttk@PC100147:~/scratch$ cat echo_script_loc.sh
#!/bin/bash

OLD_DIR="$(pwd)"
SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)

echo "pwd gives:"
echo "$OLD_DIR"

echo "script is actually in:"
echo "$SCRIPT_DIR"

(base)roberttk@PC100147:~/scratch$ source echo_script_loc.sh
pwd gives:
/Users/roberttk/scratch
script is actually in:
/Users/roberttk/scratch
(base)roberttk@PC100147:~/scratch$ cd ~
(base)roberttk@PC100147:~$ source ~/scratch/echo_script_loc.sh
pwd gives:
/Users/roberttk
script is actually in:
/Users/roberttk/scratch
```

Reply:
Indeed, I did forget that we established a location for these since I made that first comment

Reply:
I think this comment chain was never resolved, despite being marked as resolved. I'm unresolving it. We're getting wires crossed about the prod-ready way we plan to manage taxons.

- If we want to load taxons from the local clone, we shouldn't need to set this variable at all
- If the app has access to /cds, we should load from the central location stated earlier so we can update the taxons separately from the web app